src/pages/About.vue

```diff
@@ -25,19 +25,20 @@ export default class About extends Vue {
     <div class="about-page layout-constrained">
       <h1 id="main-content" tabindex="-1">About Us</h1>
 
-      <p class="constrained">
-        Electrify Chicago is an independent open-source project based out of
-        <a href="https://chihacknight.org" target="_blank"
-          >Chi Hack Night <NewTabIcon
```

Comment:
@colton-lapp - did you mean to remove this Chi Hack Night link, or was it a faulty merge?